.pre-commit-config.yaml

```diff
@@ -27,7 +27,7 @@ repos:
     hooks:
       - id: ruff
         args: [--select=F401, --fixable=F401]
-        files: ^(benchmark/|docs/|examples/)
+        files: ^(docs/|examples/)
```

Comment:
In the SGLang main, `ruff` will check `sglang/benchmark` not `sglang/sgl-kernel/benchmark` https://github.com/sgl-project/sglang/blob/main/.pre-commit-config.yaml#L30